src/Symfony/Component/Messenger/Bridge/Redis/Tests/Transport/RedisExtIntegrationTest.php

```diff
@@ -11,6 +11,7 @@
 
 namespace Symfony\Component\Messenger\Bridge\Redis\Tests\Transport;
 
+use PHPUnit\Framework\SkippedTestSuiteError;
```

Comment:
we prefer using `$this->markTestSkipped()` instead for this

Reply:
`SkippedTestSuiteError` is an internal API of PHPUnit while `markTestSkipped()` is the public API btw. So `SkippedTestSuiteError` should never be used when writing tests (in any project, not just Symfony)

Reply:
Yes, good point. I actually only copy and pasted it from the Redis tests in the Cache component in `src/Symfony/Component/Cache/Tests/Traits/RedisTraitTest.php` :sweat_smile: